src/Magnum/Audio/Context.h

```diff
@@ -41,6 +51,35 @@ typedef struct ALCcontext_struct ALCcontext;
 
 namespace Magnum { namespace Audio {
 
+class Context;
+
+/**
+@brief Run-time information about OpenAL extension
+
+Encapsulates runtime information about OpenAL extension, such as name string,
+minimal required OpenAL version and version in which the extension was adopted
+to core.
+
+See also @ref Audio::Extensions namespace, which contain compile-time information
+about OpenAL extensions.
+*/
+class MAGNUM_AUDIO_EXPORT Extension {
+    friend Context;
+
+    public:
+        /** @brief All OpenAL extensions */
+        static const std::vector<Extension>& extensions();
+
+        /** @brief Extension string */
+        constexpr const char* string() const { return _string; }
+
+    private:
+        const std::size_t _index;
+        const char* _string;
```

Comment:
Alright, thanks, I was wondering, what MSVC wanted to tell me, I was kinda guessing at this, but I didn't have time to try yet. I'll probably have some tonight.